kuksa_databroker/databroker/src/broker.rs

```diff
@@ -605,6 +615,27 @@ impl Subscriptions {
         self.change_subscriptions.push(subscription)
     }
 
+    pub fn add_continuous_subscription(
+        &mut self,
+        subscription: ContinuousSubscription,
+        frequency: u64,
+    ) {
+        let local_subscription = subscription.clone();
+        self.continuous_subscriptions
+            .lock()
+            .unwrap()
+            .push(subscription);
+
+        tokio::spawn(async move {
+            // Asynchronous code to be executed in the new task
+            while !local_subscription.sender.is_closed() {
+                let _ = local_subscription.notify(None).await;
+                // Simulate some asynchronous work
+                tokio::time::sleep(Duration::from_millis(1 / frequency * 1000)).await;
```

Comment:
How does Rust manage type conversion? `from_millis` expects a u64.  You use `1` and `1000`, will it really be a float value? Or do you rather need something like:

`Duration::from_millis((1000.0/ frequency).round() as u64)`

(I actually do not know what resolution/performance we can expect from the system, like if we request 1000 Hz, will we really receive signals with 1000 Hz?

Reply:
That would be really interesting e.g. how fast is databroker able to notify subscribers. Does it change if we have 100 subscribers at the same time. What is the limit e.g. what Erik wrote. But I guess not that much related to this PR.